torch/csrc/jit/passes/onnx/pattern_conversion/pattern_conversion.cpp

```diff
@@ -319,6 +319,7 @@ std::vector<Value*> ConvertIndexPutToONNX(
        index_put_node->input(2),
        index_put_node->input(3)});
   new_index_put_node->insertBefore(index_put_node);
+  new_index_put_node->copyMetadata(index_put_node);
```

Comment:
Should we also add `sliced_index_n->copyMetadata(insertBefore);` before[ line 50](https://github.com/pytorch/pytorch/blob/86f7c66949d72cf0186f1bc0e279e7815e62afbc/torch/csrc/jit/passes/onnx/pattern_conversion/pattern_conversion.cpp#L49) in this file?

Reply:
sure, added.